README.rst

```diff
@@ -23,7 +23,7 @@ If you'd like to join them, please consider `sponsoring Twisted's <https://docs.
 What is this?
 -------------
 
-Twisted is an event-based framework for internet applications, supporting Python 3.6+.
+Twisted is a Python event-based framework for internet applications.
```

Comment:
Or maybe you want to twiddle it a bit more.

```suggestion
Twisted is an event-based Python framework for internet applications.
```
or...
```suggestion
Twisted is an event-based framework for Python internet applications.
```
or...

ok, i'll stop now.  at this point they all feel a little bit odd now that i've thought about them too much.  `:]`

i'm also fine as-is.  change, approve, do what you will.